src/components/UserFriends/UserFriends.js

```diff
@@ -1,18 +1,46 @@
-import React, { useState } from 'react';
-import { connect } from 'react-redux';
-import mapStoreToProps from '../../redux/mapStoreToProps';
+import React, { useState } from "react";
+import { connect } from "react-redux";
+import mapStoreToProps from "../../redux/mapStoreToProps";
 
 // Basic functional component structure for React with default state
 // value setup. When making a new component be sure to replace the
 // component name TemplateFunction with the name for the new component.
 function TemplateFunction(props) {
   // Using hooks we're creating local state for a "heading" variable with
   // a default value of 'Functional Component'
-  const [heading, setHeading] = useState('Functional Component');
+  const [heading, setHeading] = useState("USER FRIENDS");
```

Comment:
Does the heading ever change? If not then it doesn't need to be state. Just set the static text in the h2.